src/cmd/report.go

```diff
@@ -142,24 +143,24 @@ func fetchGitHubData(repo string) {
 
 }
 
-func fetchDockerHubData(image string) {
+func reportDockerHubData(image string) {
```

Comment:
These functions should still be fetch as they is what they are doing. The command is reporting, not these functions.